apps/AEPSampleAppNewArchEnabled/README.md

```diff
@@ -0,0 +1,50 @@
+# Welcome to your Expo app 👋
+
+This is an [Expo](https://expo.dev) project created with [`create-expo-app`](https://www.npmjs.com/package/create-expo-app).
+
+## Get started
+
+1. Install dependencies
+
+   ```bash
+   npm install
```

Comment:
This monorepo is configured as a Yarn workspace. To install the dependencies, execute `yarn install` in the root directory.

Reply:
need to discuss this as react native versions was conflicting with each other.. both of the apps are maintaining different versions of react native

Reply:
As we discussed in the meeting, let's disable the old sample app for now.